### ISSUE TITLE
send_command predictably times out

### ISSUE THREAD

**Issue by flbuddymooreiv** [mark9064/factorio-rcon-py#1]
I think this has to do with the recv call returning but the parser not getting to its terminating condition once the response grows to a certain size. The following command succeeds with 200 iterations but fails with 300.

Here is where the timeout occurs:
https://github.com/mark9064/factorio-rcon-py/blob/59b626705797bfcfa5c0cead04f457078f04a3f3/factorio_rcon/factorio_rcon.py#L133

Here is a python test script:
```
cmd = '''/c 
  ret = ''
  for i=1,300 do
    ret = ret .. 'asdfasdf'
  end
  rcon.print(ret)
'''

srcclient = factorio_rcon.RCONClient(srchost, int(srcport), srcpass, timeout=5)
contents = srcclient.send_command(cmd)
print(contents)

```

**Comment by mark9064** [comment]
Returning large amounts of data will cause issues, though it depends where the issue lies. Factorio itself rate limits RCON (depending on a number of factors), so it could be that, or it could be a large response causing the parser to crap out. If you increase the timeout, does it fail the same amount?

**Comment by flbuddymooreiv** [comment]
`timeout=0` causes infinite hang
`timeout=10` causes 10s hang and `socket.timeout: timed out`
`timeout=20` causes 20s hang and `socket.timeout: timed out`
so on and so forth

**Comment by mark9064** [comment]
Gotcha, I'll have a look at the parser code. I did do some dirty hacks on it once to ensure that large responses always worked, as the way RCON data was sent from factorio was changed ~.40 or something and I haven't updated it since.

**Comment by flbuddymooreiv** [comment]
I see the entire response come back in `tcpdump` but the library never grabs it. 2400 bytes seems a bit restrictive.

**Comment by mark9064** [comment]
Should be fixed in 1.0.2, can you confirm it is now behaving correctly?

**Comment by flbuddymooreiv** [comment]
Confirmed
